MyToDo/App/Category+CoreDataProperties.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Category+CoreDataProperties.swift
+//  MyToDo
+//
+//  Created by Ivan Ruiz Monjo on 20/2/21.
+//
+//
+
+import Foundation
+import CoreData
+
+
+extension Category {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Category> {
+        return NSFetchRequest<Category>(entityName: "Category")
+    }
+
+    @NSManaged public var imageName: String?
+    @NSManaged public var title: String
+    @NSManaged public var todos: NSSet?
```

Comment:
Since you defined this manually, you can use `Set<ToDo>` which makes `todos` easier to use in Swift.